docs/documentation/compatibility.html

```diff
@@ -30,7 +30,7 @@ <h2 class="anchor-heading">JDK Compatibility Across Kafka Versions</h2>
                 <th>Kafka Version</th>
                 <th>Java 11</th>
                 <th>Java 17</th>
-                <th>Java 23</th>
+                <th>Java 24</th>
```

Comment:
could you please revert this change? we don't run java 24 for 4.1 and 4.0 branch

Reply:
Sure, done